packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -817,6 +829,11 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 
 		// If summarization is still unsuccessful, stop the summarizer.
 		if (status === "failure") {
+			this.mc.logger.sendErrorEvent({
+				eventName: "SummarizeFailed",
```

Comment:
Maybe use "Summarize_Failed" to be consistent with the other events like "Summarize_start", "Summarize_generate", "Summarize_op", etc.

Reply:
I prefer having a distinct event name separate from the events coming from `SummaryGenerator`